api/token.go

```diff
@@ -604,17 +613,56 @@ func (a *API) issueRefreshToken(ctx context.Context, conn *storage.Connection, u
 		if terr != nil {
```

Comment:
Just noticed this but won't `models.AddClaimToSession` always create a new row in `mfa_amr_claims` when `issueRefreshToken` is called? 

We can consider doing this when a session is created since an amr claim has to always be tied to a session

Reply:
Yup,it should always create a new row in mfa_amr_claims when `issueRefreshToken` is called. A session can have multiple AMRClaims and we make use of `issueRefreshToken` to evaluate any changes to the session.